libkineto/include/IActivityProfiler.h

```diff
@@ -59,6 +85,12 @@ class IActivityProfilerSession {
   // processes trace activities using logger
   virtual void processTrace(ActivityLogger& logger) = 0;
 
+  // returns device info used in this trace, could be nullptr
+  virtual std::unique_ptr<DeviceInfo> getDeviceInfo() = 0;
+
+  // return resource info used int this trace, could be empty
+  virtual std::vector<ResourceInfo> getResourceInfos() = 0;
```

Comment:
These two functions, getDeviceInfo and getResourceInfos have been made pure virtual, but it doesn't seem to be defined in the derived classes, CuptiRangeProfilerSession, MockProfilerSession, and GlowActivityProfilerSesison. Wouldn't this break those 3 sessions?

Reply:
Thank you, fixed it.